packages/core/package.json

```diff
@@ -4,34 +4,113 @@
+    },
+    "./cache": {
+      "types": "./dist/types/cache/index.d.ts",
+      "import": "./src/cache/index.ts"
+    },
+    "./client": {
+      "types": "./dist/types/client/index.d.ts",
+      "import": "./src/client/index.ts"
+    },
+    "./data-access": {
+      "types": "./dist/types/data-access/index.d.ts",
+      "import": "./src/data-access/index.ts"
+    },
+    "./data-access/*": {
+      "types": "./dist/types/data-access/*.d.ts",
+      "import": "./src/data-access/*.ts"
+    },
+    "./events/*": {
+      "types": "./dist/types/events/*.d.ts",
+      "import": "./src/events/*.ts"
+    },
+    "./factories": {
+      "types": "./dist/types/tests/factories/index.d.ts",
+      "import": "./src/tests/factories/index.ts"
+    },
+    "./jobs": {
+      "types": "./dist/types/jobs/index.d.ts",
+      "import": "./src/jobs/index.ts"
+    },
+    "./jobs/constants": {
+      "types": "./dist/types/jobs/constants.d.ts",
+      "import": "./src/jobs/constants.ts"
+    },
+    "./jobs/definitions": {
+      "types": "./dist/types/jobs/job-definitions/index.d.ts",
+      "import": "./src/jobs/job-definitions/index.ts"
+    },
+    "./lib/*": {
+      "types": "./dist/types/lib/*.d.ts",
+      "import": "./src/lib/*.ts"
+    },
+    "./mailers": {
+      "types": "./dist/types/mailers/index.d.ts",
+      "import": "./src/mailers/index.ts"
+    },
+    "./queues": {
+      "types": "./dist/types/jobs/queues/index.d.ts",
+      "import": "./src/jobs/queues/index.ts"
+    },
+    "./queues/types": {
+      "types": "./dist/types/jobs/queues/types.d.ts",
+      "import": "./src/jobs/queues/types.ts"
+    },
+    "./redis": {
+      "types": "./dist/types/redis/index.d.ts",
+      "import": "./src/redis/index.ts"
+    },
+    "./repositories": {
+      "types": "./dist/types/repositories/index.d.ts",
+      "import": "./src/repositories/index.ts"
+    },
+    "./repositories/*": {
+      "types": "./dist/types/repositories/*.d.ts",
+      "import": "./src/repositories/*.ts"
+    },
+    "./schema": {
+      "types": "./dist/types/schema/index.d.ts",
+      "import": "./src/schema/index.ts"
+    },
+    "./schema/*": {
+      "types": "./dist/types/schema/*.d.ts",
+      "import": "./src/schema/*.ts"
+    },
+    "./services/*": {
+      "types": "./dist/types/services/*.d.ts",
+      "import": "./src/services/*.ts"
+    },
+    "./test": {
+      "types": "./dist/types/tests/useTestDatabase.d.ts",
+      "import": "./src/tests/useTestDatabase.ts"
+    },
+    "./websockets/*": {
+      "types": "./dist/types/websockets/*.d.ts",
+      "import": "./src/websockets/*.ts"
+    },
+    "./workers/*": {
+      "types": "./dist/types/workers/*.d.ts",
+      "import": "./src/workers/*.js"
+    },
+    "./utils/*": {
+      "types": "./dist/types/utils/*.d.ts",
+      "import": "./src/utils/*.ts"
+    },
+    "./telemetry": {
+      "types": "./dist/types/telemetry/index.d.ts",
+      "import": "./src/telemetry/index.ts"
+    }
```

Comment:
tell typescript where to find the type declarations

Reply:
Type declarations on a TypeScript code base consumed by an app make in Typescript